canonical/store-components
# PackageCard bundled charms

## Done
Added bundle icons to PackageCard component

## QA
- Go to https://store-components-12.demos.haus/?path=/story/packagecard--bundle
- Check that there are icons for bundled charms

## Issue
- Fixes https://warthogs.atlassian.net/browse/WD-1627
- Fixes https://warthogs.atlassian.net/browse/WD-1628
